### PR TITLE
add doc for data source

### PR DESCRIPTION
**Pull request by outscale-hmi**

### CODE REVIEW

File: docs/datasources/omi.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+---
+
+# Outscale OMI Data Source
+
+Type: `outscale-omi`
+
+The Outscale OMI data source will filter and fetch an Outscale OMI, and output all the OMI information that will
+be then available to use in the [Outscale builders](https://developer.hashicorp.com/packer/plugins/builders/outscale).
+
+-> **Note:** Data sources is a feature exclusively available to HCL2 templates.
```
Comment (outscale-mdr) [inline docs/datasources/omi.mdx:17]: "Data source"

File: docs-partials/datasource/omi/DatasourceOutput.mdx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!-- Code generated from the comments of the DatasourceOutput struct in datasource/omi/data.go; DO NOT EDIT MANUALLY -->
+
+- `id` (string) - The ID of the AMI.
+
+- `name` (string) - The name of the AMI.
+
+- `creation_date` (string) - The date of creation of the AMI.
+
+- `owner` (string) - The AWS account ID of the owner.
```
Comment (outscale-mdr) [inline docs-partials/datasource/omi/DatasourceOutput.mdx:9]: Outscale account Id

File: docs-partials/datasource/omi/DatasourceOutput.mdx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<!-- Code generated from the comments of the DatasourceOutput struct in datasource/omi/data.go; DO NOT EDIT MANUALLY -->
+
+- `id` (string) - The ID of the AMI.
```
Comment (outscale-mdr) [inline docs-partials/datasource/omi/DatasourceOutput.mdx:3]: OMI

**Review by outscale-mdr — CHANGES_REQUESTED** [review]

File: docs/datasources/omi.mdx (hunk 1 above)
Comment (outscale-hmi) [reply docs/datasources/omi.mdx:17]: done

File: docs/datasources/omi.mdx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    filters = {
+        virtualization-type = "hvm"
+        name = "ubuntu/images/*ubuntu-xenial-16.04-amd64-server-*"
+        root-device-type = "ebs"
+    }
+    owners = ["099720109477"]
+    most_recent = true
+}
+```
+This selects the most recent Ubuntu 16.04 HVM EBS AMI from Canonical. Note that the data source will fail unless
```
[the remark this answers is not in the document]
Comment (outscale-hmi) [reply docs/datasources/omi.mdx]: done

File: docs-partials/datasource/omi/DatasourceOutput.mdx (hunk 2 above)
Comment (outscale-hmi) [reply docs-partials/datasource/omi/DatasourceOutput.mdx]: done

File: docs-partials/datasource/omi/DatasourceOutput.mdx (hunk 3 above)
Comment (outscale-hmi) [reply docs-partials/datasource/omi/DatasourceOutput.mdx]: done

**Review by outscale-mdr — APPROVED** [review]

**Pull request merged**